### ISSUE TITLE
Halloween Costume

### ISSUE THREAD

**Issue by 3dphoto** [s1pierro/Papier#41]
How to create for larger than A4/Letter? 
Would this be possible with Papier?

![image](https://user-images.githubusercontent.com/28678758/41268096-6abb358a-6db3-11e8-86dc-7c405644969f.png)

![image](https://user-images.githubusercontent.com/28678758/41268112-7a561fbe-6db3-11e8-89ef-d43e8ab43637.png)

![image](https://user-images.githubusercontent.com/28678758/41268124-88db84b6-6db3-11e8-8aa5-10289f53dd4d.png)

![image](https://user-images.githubusercontent.com/28678758/41268134-924b7a7e-6db3-11e8-947b-6d34b0ebc426.png)

![image](https://user-images.githubusercontent.com/28678758/41268165-b299ea04-6db3-11e8-96ca-cf4c26c6ac88.png)

![image](https://user-images.githubusercontent.com/28678758/41268175-bc6af654-6db3-11e8-838d-c7f7a5ad8159.png)

![image](https://user-images.githubusercontent.com/28678758/41268199-d66d9a2a-6db3-11e8-94ae-b5823cdab8e1.png)

![image](https://user-images.githubusercontent.com/28678758/41268222-ee5b091a-6db3-11e8-82b8-9254282a4494.png)

**Comment by s1pierro** [comment]
Yes it will be possible. You can already have an overview of that feature in the pattern ( island ? ) view ( left bottom ). The app emulate sizes A3, A2, A1, A0 to fit the pattern ( island ?)  requirement. Marcks will be add to the final A4 documents to easy assemble them. By using transparency with  help of a window for ex.

**Comment by s1pierro** [comment]
![capture d ecran de 2018-06-13 13-38-08](https://user-images.githubusercontent.com/6394332/41349171-5067d712-6f0f-11e8-893f-ffbb43527712.png)
![capture d ecran de 2018-06-13 13-38-46](https://user-images.githubusercontent.com/6394332/41349184-5f5d64a8-6f0f-11e8-8c2c-e5ca34983ea7.png)
![capture d ecran de 2018-06-13 13-38-57](https://user-images.githubusercontent.com/6394332/41349203-6ef9ca14-6f0f-11e8-828a-af2454d399ec.png)
![capture d ecran de 2018-06-13 13-39-20](https://user-images.githubusercontent.com/6394332/41349209-70b99910-6f0f-11e8-87ea-7f0830ca6d11.png)

**Comment by s1pierro** [comment]
those three past days, I'have been improving the patterns auto layout feature. it can and it will be improved much more, but as it is, I ear it acceptable. The app now dispatch by itself the 24 paperAce body patterns into 8 A4. 20 in past.


Now I hesitate for the next one. As I am launched, I could continue with the print view of the final document. But it's a little useless as long as the tongue/tab assembly system is missing.

**Comment by 3dphoto** [comment]
The preview you are sharing is impressive. I'm coming up with ideas with A4 on how to manage as well.

Thinking a safe bet would be to create a **divider bar** that the user can **slide left to right** the A4 would **grow and shrink** depending on how much screen space a user would allow and/or **collapse** this area to only focus on the model with Papier.

In the same sense the Three.js model could grow or shrink, you could also provide a toggle option to leave static while working with the divider bar.

https://dribbble.com/shots/3997043-Watering-Tracker-App/attachments/921018 = vertical concept of model on top while A4 is at the bottom
http://www.mikevalera.com/wp-content/uploads/mobile-gbox-open-1.6.jpg

---

**Reference**
https://github.com/s1pierro/Papier/issues/14 [Desktop divide and conquer + scaling of A4 paper](https://github.com/s1pierro/Papier/issues/14)

**Comment by s1pierro** [comment]
![capture d ecran de 2018-06-13 16-41-07](https://user-images.githubusercontent.com/6394332/41358422-a1ab8132-6f28-11e8-962e-1822cb94494a.png)

**Comment by 3dphoto** [comment]
What program are you making your obj(s) with?
Blender, Maya or some other program?

**Comment by 3dphoto** [comment]
Good to know! 
Thanks for sharing!
